### PR TITLE
New Port: Hawk Island

### PR DESCRIPTION
**Pull request by Ganimoth**
# New Port for Hawk Island

## Game Information
- **Title**: Hawk Island
- **URL**: https://lutd.itch.io/hawk-island

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x ] AmberELEC
- [x ] ArkOS
- [x ] ROCKNIX
- [x ] muOS

### Resolution Tests
Test all major resolutions:
- [x ] 480x320
- [x ] 640x480
- [ ] 720x720 (RGB30)
- [x ] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

**Comment by Ganimoth** [comment]
As I just noticed in another ongoing PR, do we still need printf "\033c" > /dev/tty0?

**Comment by Cebion** [comment]
> As I just noticed in another ongoing PR, do we still need printf "\033c" > /dev/tty0?

Not anymore

**Review by Cebion — DISMISSED** [review]

File: ports/hawkisland/Hawk Island.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# We log the execution of the script into log.txt
+> "$GAMEDIR/log.txt" && exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+$GPTOKEYB "gmloadernext" &
+pm_platform_helper "$GAMEDIR/gmloadernext"
+
+$ESUDO chmod +x "$GAMEDIR/gmloadernext"
+
+./gmloadernext game.apk
```
Comment (cdeletre) [inline ports/hawkisland/Hawk Island.sh:34]: gmloadernext doesn't use this command line argument to load the apk, it's always game.apk. (see [here](https://github.com/JohnnyonFlame/gmloader-next/blob/6f2569914109866c1fb903a9b7aed970896b5618/gmloader/main.cpp#L83))
```suggestion
./gmloadernext
```

File: ports/hawkisland/hawkisland/game.apk
Comment (cdeletre) [inline ports/hawkisland/hawkisland/game.apk:1]: The file needs some cleaning, please remove:
- AndroidManifest.xml
- androidsupportmultidexversion.txt
- classes.dex
- META-INF/*
- res/*
- resources.arsc

To do so you can run
`zip -d game.apk AndroidManifest.xml androidsupportmultidexversion.txt classes.dex "res/*" "META-INF/*  resources.arsc"`

I let you check that the game still runs but it should be ok.

**Review by cdeletre — CHANGES_REQUESTED** [review]

File: ports/hawkisland/hawkisland/game.apk
Comment (Ganimoth) [reply ports/hawkisland/hawkisland/game.apk:1]: Done, game seems to be still working fine.

**Review by Cebion — DISMISSED** [review]

**Review by cdeletre — DISMISSED** [review]

**Review by Cebion — APPROVED** [review]

**Review by cdeletre — APPROVED** [review]

**Pull request merged**